### PR TITLE
Update the plugin version and date automatically

### PR DESCRIPTION
**Pull request by ryantxu**
use the package.json version and current date in plugin.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "copy-webpack-plugin": "^4.5.1",
     "css-loader": "^0.28.11",
     "grafana-sdk-mocks": "github:grafana/grafana-sdk-mocks",
+    "replace-in-file-webpack-plugin": "^1.0.6",
```
Comment (davkal) [inline package.json]: This needs to be filed in alphabetical order.

File: webpack.config.js [hunk 2]
```diff
@@ -37,6 +41,17 @@ module.exports = {
...
     ]),
+    new ReplaceInFileWebpackPlugin([{
+      dir: 'dist',
+      files: ['plugin.json'],
+      rules: [{
+        search: '%VERSION%',
+        replace: packageJson.version
+      },{
+        search: '%TODAY%',
+        replace: moment().format('YYYY.MM.DD')
```
Comment (davkal) [inline webpack.config.js]: Please use ISO8601 here, and refrain from using `moment`, we might replace it soon.

File: package.json (hunk 1 above)
Comment (ryantxu) [reply package.json]: done

**Comment by davkal** [comment]
Changes look good! Could you please rebase and fix conflict again?

(Also, do you have owner push enabled? I got permission denied when trying to push a new yarn.lock.)

**Comment by ryantxu** [comment]
@davkal -- merged with master, should be good now.  owner push enabled, I have:
ppp
anything else?

**Pull request merged**